docs/BreakingChanges/116 GridViews with AllowCustomPaging set to true may fire the PageIndexChanging event when leaving the final page of the view.md

```diff
@@ -0,0 +1,36 @@
+## 116: GridViews with AllowCustomPaging set to true may fire the PageIndexChanging event when leaving the final page of the view
```

Comment:
{
  "Id": "116",
  "Title": "GridViews with AllowCustomPaging set to true may fire the PageIndexChanging event when leaving the final page of the view",
  "Details": "A bug in the .NET Framework 4.5 causes `PageIndexChanging` to sometimes not fire for GridViews that have enabled `AllowCustomPaging`",
  "Suggestion": "This issue has been fixed in the .NET Framework 4.6 and may be addressed by upgrading to that version of the .NET Framework. As a work-around, the app can do an explicit BindGrid on any `Page_Load` that would hit these conditions (the GridView is on the last page and LastPageSize is different from PageSize). Alternatively, the app can be modified to allow paging (instead of custom paging), as that scenario does not demonstrate the problem.",
  "Link": "http://stackoverflow.com/questions/18793795/asp-net-4-5-gridview-postback-of-last-page",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Web.UI.WebControls.GridView.set_AllowCustomPaging(System.Boolean)",
    "P:System.Web.UI.WebControls.GridView.AllowCustomPaging"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.6",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/782891",
  "Notes": "An analyzer could usefully scan XAML (where custom paging is more likely to be enabled)\nSource analyzer status: Pri 3, Planned - No tracking work item yet",
  "IsRetargeting": false,
  "ImpactScope": 2
}

**Parsed APIs:**
- [x] M:System.Web.UI.WebControls.GridView.set_AllowCustomPaging(System.Boolean)
- [x] P:System.Web.UI.WebControls.GridView.AllowCustomPaging